js/ui.js

```diff
@@ -0,0 +1,586 @@
+        }
+    }
+}
+
+function setStickySidebar(sticky) {
+    gameData.settings.stickySidebar = sticky;
+    settingsStickySidebar.checked = sticky;
+    info.style.position = sticky ? 'sticky' : 'initial';
+}
+
+function selectElementInGroup(group, index) {
+    const elements = document.getElementsByClassName(group)
+    for (const el of elements) {
+        el.classList.remove("selected")
+    }
+    elements[index].classList.add("selected")
+}
+
+function setLayout(id) {
+    gameData.settings.layout = id
+    if (id == 0) {
+        document.getElementById("skillsTabButton").classList.add("hidden")
+        document.getElementById("shopTabButton").classList.add("hidden")
+
+        document.getElementById("dummyPage1").classList.remove("hidden")
+        document.getElementById("dummyPage2").classList.remove("hidden")
+        document.getElementById("dummyPage3").classList.remove("hidden")
+
+        document.getElementById("skills").classList.add("hidden")
+        document.getElementById("shop").classList.add("hidden")
+
+        document.getElementById("tabcolumn").classList.add("plain-tab-column")
+        document.getElementById("tabcolumn").classList.remove("tabs-tab-column")
+
+        document.getElementById("maincolumn").classList.add("plain-main-column")
+        document.getElementById("maincolumn").classList.remove("tabs-main-column")
+
+        document.getElementById("jobs").appendChild(document.getElementById("skillPage"))
+        document.getElementById("jobs").appendChild(document.getElementById("itemPage"))
+
+    } else {
+        document.getElementById("skillsTabButton").classList.remove("hidden")
+        document.getElementById("shopTabButton").classList.remove("hidden")
+
+        document.getElementById("dummyPage1").classList.add("hidden")
+        document.getElementById("dummyPage2").classList.add("hidden")
+        document.getElementById("dummyPage3").classList.add("hidden")
+
+        document.getElementById("skills").classList.remove("hidden")
+        document.getElementById("shop").classList.remove("hidden")
+
+        document.getElementById("tabcolumn").classList.add("tabs-tab-column")
+        document.getElementById("tabcolumn").classList.remove("plain-tab-column")
+
+        document.getElementById("maincolumn").classList.add("tabs-main-column")
+        document.getElementById("maincolumn").classList.remove("plain-main-column")
+
+        document.getElementById("skills").appendChild(document.getElementById("skillPage"))
+        document.getElementById("shop").appendChild(document.getElementById("itemPage"))
+    }
+
+    selectElementInGroup("Layout", id == 0 ? 1 : 0)
+}
+
+function setFontSize(id) {
+    const fontSizes = {
+        0: "xx-small",
+        1: "x-small",
+        2: "small",
+        3: "medium",
+        4: "large",
+        5: "x-large",
+        6: "xx-large",
+        7: "xxx-large",
+    }
+
+    if (id < 0) id = 0
+    if (id > 7) id = 7
```

Comment:
This could be rewritten in one line as follows:
```js
id = Math.max(0, Math.min(7, id))
```

Reply:
I like the previous aproach more.